keys.go

```diff
@@ -56,6 +56,9 @@ type OldVerifyKey struct {
 	ExpiredTS Timestamp `json:"expired_ts"`
 }
 
+// A KeyID is the ID of a ed2219 key used by a server.
```

Comment:
Is this ID arbitrary? Does it have a particular structure like `ed25519:randomtext`? Can `randomtext` contain :hankey: ? ASCII only? Spaces?